gef.py

```diff
@@ -4735,11 +4735,11 @@ def ps(self):
             fields = line.split()
             t = {}
 
-            for i in range(len(names)):
+            for i, name in enumerate(names):
```

Comment:
12 years doing Python code, and I am just learning `enumerate()`, I feel like a noob, it's awesome. I like that, very Pythonic.